tests/pytests/test_vecsim.py

```diff
@@ -608,6 +608,14 @@ def test_with_fields():
         #  KNN query instead of 100 (but not for range) - should be fixed
         env.assertEqual(res[1:], res_range[1:])
 
+    # MOD-7887 - bad error message
+    # Expect to get the same result when using a score field and when not, in case of a field name that does not exist in the schema
+    exp = env.expect('FT.SEARCH', 'idx', '*=>[KNN 100 @not_a_field $vec_param]', 'PARAMS', 2, 'vec_param', query_data.tobytes()).res
```

Comment:
also we want to get the same output as if it wasnt a vecsim query (@t:"meow" or something) that includes a field doesn't exist in the schema